README.md

```diff
@@ -224,29 +242,24 @@ cd che
 git checkout openshift-connector
 ```
 
-* Set some environment variables (or run `. ./scripts/setenv-for-build.sh`)
+* Set some environment variables:
+    * `UPSTREAM_CHE_PATH` points to the local directory where the official Che repo was checked out
+    and is *mandatory* for some scripts
 
 ```bash
-export CHE_IMAGE_REPO=rhche/che-server
-export CHE_IMAGE_TAG=nightly
```

Comment:
The name and tag of the docker images we want to push to Minishift is now automatically set in the build and deploy scripts according to:
- the variable `DOCKER_HUB_NAMESPACE`
- the fact that the we build *with or without* the dashboard (thanks to the `-DwithoutDashboard` build argument)

The code can be see here for example: https://github.com/davidfestal/rh-che/blob/f3c7bfbafcaacd9f8e8920b578ed1a00af166582/dev-scripts/build_fabric8_and_deploy.sh#L18